src/lib.rs

```diff
@@ -63,6 +63,18 @@ pub enum Atom {
     // it might happen that it contains several leaves.
     DeleteBegin,
     DeleteEnd,
+    /// Scoped commands
+    // ScopedSoftline works together with the @open_scope and @end_scope query tags.
+    // To decide if a scoped softline must be expanded into a hardline, we look at
+    // the innermost scope earing the corresponding `scope_id`, that encompasses it.
```

Comment:
Minor: "earing" is not right. I _think_ (I'm not sure) you mean this:
```suggestion
    // the innermost scope having the corresponding `scope_id` that encompasses it.
```